sdcm/nemesis.py

```diff
@@ -966,7 +969,12 @@ def generate_random_parameters_values():
 
         args = generate_random_parameters_values()
         sub_cmd_args = "{ks} {cf} {duration} -s {capacity} -k {toppartition} -a {samplers}".format(**args)
-        result = self.target_node.run_nodetool(sub_cmd='toppartitions', args=sub_cmd_args)
+        try:
+            result = self.target_node.run_nodetool(sub_cmd='toppartitions', args=sub_cmd_args)
```

Comment:
`run_nodetool` has `verbose=True` by default that is passed to remote runner that print this information...